src/stdatamodels/fits_support.py

```diff
@@ -155,6 +156,9 @@ def get_hdu(hdulist, hdu_name, index=None):
 
 
 def _make_hdu(hdulist, hdu_name, index=None, hdu_type=None, value=None):
+    if isinstance(value, NDArrayType):
```

Comment:
@kmacdonald-stsci did you check the update case?  I mean, when the HDU doesn't need to be created and the value is assigned to an existing HDU?

Reply:
I checked this and it works fine.